pkg/util/test/util.go

```diff
@@ -2337,13 +2351,31 @@ func ValidateTelemetryUninstalled(pxImageList map[string]string, cluster *corev1
 
 // ValidateTelemetry validates telemetry component is installed/uninstalled as expected
 func ValidateTelemetry(pxImageList map[string]string, cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
+	logrus.Info("Validate Telemetry components")
+	logrus.Info("Check PX and PX Operator versions to determine which telemetry to validate against..")
+	pxVersion := GetPortworxVersion(cluster)
+	fmt.Printf("PX Version: %v", pxVersion)
+	opVersion, err := GetPxOperatorVersion()
+	if err != nil {
+		return err
+	}
+	logrus.Infof("PX Operator version: %v", opVersion)
+
 	if cluster.Spec.Monitoring != nil &&
 		cluster.Spec.Monitoring.Telemetry != nil &&
 		cluster.Spec.Monitoring.Telemetry.Enabled {
+		logrus.Info("Telemetry is enabled in StorageCluster")
+		if pxVersion.GreaterThanOrEqual(pxVer2_12) && pxVersion.GreaterThanOrEqual(opVer1_9_1) {
+			return ValidateTelemetryEnabled(pxImageList, cluster, timeout, interval)
```

Comment:
are you implementing a new validation function for ccm-go only? that case can we call it `ValidateTelemetryV2Installed` and change the old one to `ValidateTelemetryV1Installed` to be more clear?

Reply:
done